src/foremast/utils/templates.py

```diff
@@ -25,7 +26,7 @@
 LOG = logging.getLogger(__name__)
 
 HERE = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Convert this to `pathlib` as well, for consistency.

Reply:
Good idea. I did it.